src/components/InputDataDialog.spec.js

```diff
@@ -0,0 +1,50 @@
+import {render, screen} from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
+
+import InputDataDialog from './InputDataDialog.js';
+
+describe('InputDataDialog', () => {
+
+    it('is an accessible form', () => {
```

Comment:
For your tests, please have a look at the tests Stefan showed you :) 
Additionally, as your input field is required, I would also test that.